go/go.mod

```diff
@@ -2,13 +2,12 @@ module github.com/cucumber/gherkin/go/v26
 
 require (
 	github.com/cucumber/messages/go/v21 v21.0.1
-	github.com/cucumber/messages/go/v22 v22.0.0
+	github.com/gofrs/uuid v4.3.1+incompatible
```

Comment:
I'm not very familiar with Go. Why did this become a direct dependency?

Reply:
Likely because I imported it for [this benchmark](https://github.com/cucumber/gherkin/blob/5b07ef65842033b2a232e18c651f2841d1c17ac1/go/matcher_test.go#L7)

The benchmarks aren't required for this merge so I can just remove them if you aren't happy with this being direct.

Reply:
Yes, please!

Reply:
I have a proper fix for that dependency upgrade in #133.